packages/stargate/src/Stargate.ts

```diff
@@ -0,0 +1,67 @@
+import { type BridgeActionParams, compressJson } from '@rabbitholegg/questdk'
+import { type Address } from 'viem'
+import { STARGATE_BRIDGE_ABI } from './abi.js'
+import {
+  CHAIN_ID_ARRAY,
+  LAYER_ZERO_TO_LAYER_ONE_CHAIN_ID,
+} from './chain-ids.js'
+import {
+  CHAIN_AND_POOL_TO_TOKEN_ADDRESS,
+  CHAIN_ID_TO_ROUTER_ADDRESS,
+  CHAIN_ID_TO_ETH_ROUTER_ADDRESS,
+} from './contract-addresses.js'
+
+// If you're implementing swap or mint, simply duplicate this function and change the name
+export const bridge = async (bridge: BridgeActionParams) => {
+  // This is the information we'll use to compose the Transaction object
+  const {
+    sourceChainId,
+    destinationChainId,
+    contractAddress,
+    tokenAddress,
+    amount,
+    recipient,
+  } = bridge
+  const layerOneChainId = LAYER_ZERO_TO_LAYER_ONE_CHAIN_ID[sourceChainId]
+  const sourcePool = tokenAddress
+    ? CHAIN_AND_POOL_TO_TOKEN_ADDRESS[sourceChainId][tokenAddress]
+    : 0
+
+  if (sourcePool === 13) {
```

Comment:
is this `const ETH_POOL = 13` accurate?

Reply:
Yep, I'll create a const and compare against that vs a magic number for increased clarity.